### DISCUSSION TITLE
Custom Instructions From Repo

### DISCUSSION THREAD

**Discussion by emshvac** [cline/cline#586] [Feature Requests]
I am finding myself having to keep a file in each of my repos.  Right now it is just custominstructions.txt.  It would be nice if we could specify this in the cline config.  I find that for each project I work on I am having to make custom instructions in order the the LLM to do things correctly.

IE:  Anytime you need to log something please ensure to use my logging library located XXX/XXX.xx and make sure to implement a relevant tag.

**Comment by heltonteixeira** [comment]
You can write these instructions on the custom instructions in the settings.

**Comment by sgryphon** [comment]
This feature already exists via clinerules configuration. https://docs.cline.bot/features/cline-rules

I also have a pull request up for also supporting a standard (cross-agent) AGENTS.md file https://github.com/cline/cline/pull/5405
